docs/fastdds/dds_layer/core/policy/eprosimaExtensions.rst

```diff
@@ -1020,6 +1026,10 @@ List of QoS Policy data members:
   be changed using this data member.
 * |TransportConfigQos::builtin_transports_reception_threads-api|:
   The |ThreadSettings| for the reception threads of the builtin transports.
+* |TransportConfigQos::max_msg_size_no_frag-api|:
+  Maximum message size used to avoid fragmentation, set only in LARGE_DATA.
```

Comment:
```suggestion
  Maximum message size used to avoid fragmentation.
  Useful when the configured transports allow for big datagrams to be sent (i.e. SHM or TCP).
```